modules/Clinical/domains/demographics.yaml

```diff
@@ -57,27 +45,43 @@ enums:
       "Unknown":
         description: Not known, not observed, not recorded, or refused
 
+  SexEnum:
```

Comment:
This is probaly for us to go back to Kristen on... but we should expand our sax descriptions here given we also have gender... is this sex assigned at birth? is it self reported sex?

Reply:
I guess this is in CADSR. per the slot below